### DISCUSSION TITLE
Table with over 255 columns fails to bind values

### DISCUSSION THREAD

**Discussion by lachlanorr** [aarondl/sqlboiler#1180] [Q&A] [answered]
I'm trying to model a table with 259 columns. It is a legacy rails DB that we're modernizing, and for now I'm stuck with this reality.

I noticed if I take out 4 of the columns it works, but over 255 results in a bind error.

I think it has to do with this line of code:
https://github.com/volatiletech/sqlboiler/blob/master/queries/reflect.go#L337

The sentinel value is 255, and the & operation is preventing more than 255 columns I think.

I was hoping someone familiar with the code could comment on this and help me understand if there is a workaround or possible enhancement to sqlboiler that may overcome the problem.

**Comment by stephenafamo** [comment, marked as answer]
I believe it would be possible to refactor and make it possible to bind more than 255 results, an attempt was made earlier in #911 

Perhaps you can create a fork with these changes and see if it works for you. I cannot guarantee that it will be merged into the main repo, at least not until we fully understand any other side effects.
